src/javatests/com/google/cloud/bigquery/dwhassessment/extractiontool/db/ScriptManagerImplTest.java

```diff
@@ -44,37 +53,49 @@
 @RunWith(JUnit4.class)
 public final class ScriptManagerImplTest {
 
+  private final String baseScript = "SELECT * FROM TestTable";
+
   private final ImmutableMap<String, Supplier<String>> scriptsMap =
-      ImmutableMap.of("default", () -> "SELECT * FROM TestTable");
+      ImmutableMap.of(
+          "default",
+          () -> baseScript,
+          "default_chunked",
+          () ->
+              baseScript
+                  + "{{#if sortingColumns}}\n"
+                  + "ORDER BY {{#each sortingColumns}}{{this}}{{#unless"
+                  + " @last}},{{/unless}}{{/each}} ASC NULLS FIRST\n"
+                  + "{{/if}}");
+  private final ImmutableMap<String, ImmutableList<String>> sortingColumnsMap =
+      ImmutableMap.of("default_chunked", ImmutableList.of("TIMESTAMPS"));
```

Comment:
from what I understand this actually is used only in `executeScript_writeChunked_success`. 
If it's true, it can be safely empty most of the time

Reply:
This map being non-empty doesn't cause much overhead but can also verify that it does not interfere with other non-chunking scripts' executions.

Reply:
I don't follow how does it interfere, It is a no-op for all non-chunking executions:
```
ImmutableList<String> sortingColumns = chunkMode ? sortingColumnsMap.get(scriptName) : ImmutableList.of();
```

Reply:
Indeed it is, with the current implementation details. Testing is also for when such details are changed.